src/main/java/net/minecraftforge/fmp/util/MCMPEventHandler.java

```diff
@@ -0,0 +1,56 @@
+package net.minecraftforge.fmp.util;
+
+import java.util.Map.Entry;
+
+import net.minecraft.block.properties.IProperty;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.Minecraft;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.util.text.TextFormatting;
+import net.minecraftforge.client.event.RenderGameOverlayEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+import net.minecraftforge.fmp.multipart.MultipartRegistry;
+
+public class MCMPEventHandler
```

Comment:
There's no real need for that... It doesn't hurt anybody. It's been moved since the last commit, though. It's in net.minecraftforge.fmp and the util package is gone.